src/main/java/org/advent/dayten/CoordinateDirection.java

```diff
@@ -0,0 +1,79 @@
+package org.advent.dayten;
+
+public enum CoordinateDirection {
+
+    UP(0, -1),
+    DOWN(0, 1),
+    LEFT(-1, 0),
+    RIGHT(1, 0);
+
+    private final int x;
+    private final int y;
+
+    CoordinateDirection(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public static CoordinateDirection getDirection(char c, Position position, int x, int y) {
```

Comment:
rethink this, it doesnt need to be static. 
maybe the enum should be talking about the characters instead of the directions and rather link 2 directions via the character